filebrowser/main.tf

```
@@ -14,6 +14,24 @@ variable "agent_id" {
   description = "The ID of a Coder agent."
 }
 
+variable "workspace_name" {
+  type = string
+  default = ""
+  description = "Set this and owner_name to serve filebrowser from subdirectory."
+}
+
+variable "owner_name" {
+  type = string
+  default = ""
+  description = "Set this and workspace_name to serve filebrowser from subdirectory."
+}
+
+variable "resource_name" {
+  type = string
+  default = "main"
+  description = "The name of the main deployment. (Used to build the subdirectory of the module.)"
+}
```

Lets call it `agent_name` we 
```suggestion
variable "agent_name" {
  type = string
  default = ""
  description = "The name of the main deployment. (Used to build the subpath for coder_app.)"
}
```